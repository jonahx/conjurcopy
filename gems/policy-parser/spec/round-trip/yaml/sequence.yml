- !group developers
- !user
  id: alice