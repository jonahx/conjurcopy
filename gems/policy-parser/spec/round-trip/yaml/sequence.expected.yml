---
- !group
  id: developers
- !user
  id: alice
